patches/all/@oclif+dev-cli+1.22.2.patch

```diff
@@ -1,3 +1,18 @@
+diff --git a/node_modules/@oclif/dev-cli/lib/commands/pack/deb.js b/node_modules/@oclif/dev-cli/lib/commands/pack/deb.js
+index 51bce6e..a38f00f 100644
+--- a/node_modules/@oclif/dev-cli/lib/commands/pack/deb.js
++++ b/node_modules/@oclif/dev-cli/lib/commands/pack/deb.js
+@@ -37,8 +37,8 @@ class PackDeb extends command_1.Command {
+             await qq.write([workspace, 'DEBIAN/control'], scripts.control(buildConfig, debArch(arch)));
+             await qq.chmod([workspace, 'usr/lib', config.dirname, 'bin', config.bin], 0o755);
+             await qq.x(`ln -s "../lib/${config.dirname}/bin/${config.bin}" "${workspace}/usr/bin/${config.bin}"`);
+-            await qq.x(`chown -R root "${workspace}"`);
+-            await qq.x(`chgrp -R root "${workspace}"`);
++            await qq.x(`sudo chown -R root "${workspace}"`);
++            await qq.x(`sudo chgrp -R root "${workspace}"`);
```

Comment:
Just wondering:
* If `npm run package` was executed as the root user, then I think that adding `sudo` here is a no-op.
* It it was not executed as the root user, then couldn't this cause `sudo` to interactively ask for a password? Thus blocking the build...